### PR TITLE
Add GA event tracking for actions in trace view

### PR DESCRIPTION
**Pull request by tiffon**
Partially addresses #157.

Adds GA event tracking for trace view. 

This adds events for:

- Viewing time range
- Filter set / clear
- Open / close
    - The trace header
    - Parents
    - Span details
    - Tags, process, logs, log-items in span details
- Resize the span-name column

Search and dependency graphs are outstanding and will likely be a different PR.

### CODE REVIEW

**Comment by tiffon** [comment]
@simonrobb Can you look this PR over when you get the chance?

File: src/components/TracePage/KeyboardShortcutsHelp.track.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { trackEvent } from '../../utils/tracking';
+
+const context = 'jaeger/ux/trace/kbd-modal';
+
+export default function trackKbdHelpModal() {
```
Comment (simonrobb) [inline src/components/TracePage/KeyboardShortcutsHelp.track.js]: Given you're tracking an event maybe be more verbose with the event being tracked. Suggestion: `trackKbdHelpModalOpen`

File: src/components/TracePage/TracePageHeader.track.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { trackEvent } from '../../utils/tracking';
+
+const altViewCtx = 'jaeger/ux/trace/alt-view';
+export const slimHeaderCtx = 'jaeger/ux/trace/slim-header';
+
+export function trackAltView() {
```
Comment (simonrobb) [inline src/components/TracePage/TracePageHeader.track.js]: Suggestion: `trackAltViewOpen`

File: src/components/TracePage/TracePageHeader.track.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  trackEvent({
+    category: altViewCtx,
+    action: 'open',
+  });
+}
+
+export function trackSlimHeader(isOpen: boolean) {
```
Comment (simonrobb) [inline src/components/TracePage/TracePageHeader.track.js]: Suggestion: `trackSlimHeaderStateChange`

File: src/index.js [hunk 4]
```diff
@@ -16,9 +16,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { document } from 'global';
 
-import JaegerUIApp from './components/App';
-import { context as trackingContext } from './utils/tracking';
-
+// these need to go after the App import
```
Comment (simonrobb) [inline src/index.js]: Before or after?

Comment (tiffon) [reply src/index.js]: Doh! Merge conflicts... Thanks!

File: src/components/TracePage/TracePageHeader.track.js (hunk 3 above)
Comment (tiffon) [reply src/components/TracePage/TracePageHeader.track.js]: :+1:

File: src/components/TracePage/TracePageHeader.track.js (hunk 2 above)
Comment (tiffon) [reply src/components/TracePage/TracePageHeader.track.js]: :+1:

File: src/components/TracePage/KeyboardShortcutsHelp.track.js (hunk 1 above)
Comment (tiffon) [reply src/components/TracePage/KeyboardShortcutsHelp.track.js]: :+1:

**Pull request merged**